Testing/Stereo-Inertial/run_stereo_inertial_tests.sh

```diff
@@ -0,0 +1,23 @@
+echo "Running tests for Stereo-Inertial SLAM!"
```

Comment:
1. 1st line should change working directory to the directory the script is in.
2. All paths should be relative (including the dataset paths